itou/templates/search/siaes_search_home.html

```diff
@@ -1,68 +1,99 @@
 {% extends "layout/base.html" %}
 {% load redirection_fields %}
+{% load theme_inclusion %}
 {% load static %}
 
 {% block title %}Les emplois de l'inclusion{% endblock %}
 
+{% block body_class %}p-home{% endblock %}
+
 {% block content_title_wrapper %}{% endblock %}
 
 {% block content %}
 
     {% include "search/includes/generic_search_home.html" with form=siae_search_form title="Rechercher un emploi inclusif" url_search="search:employers_results" template_form_name="search/includes/siaes_search_form.html" %}
 
-    {% if not user.is_authenticated %}
-        <section class="bg-white text-center pt-5 pb-6">
-            <div class="container">
-                <h2 class="h1">Nos missions</h2>
-                <p class="lead">
-                    Ce service facilite la mise en relation des personnes les plus éloignées de l'emploi avec les employeurs solidaires (SIAE, GEIQ, EA et EATT) et les accompagnants (orienteurs et prescripteurs habilités).
-                    <br>
-                    Il offre aux utilisateurs un outil mutualisé pour simplifier les procédures, fluidifier les parcours d'insertion entre professionnels et renforcer la qualité de l'accompagnement des personnes.
-                </p>
+    <section class="s-article-figure-01 s-article-figure-01--ltr">
+        <div class="s-article-figure-01__container container container-max-lg">
+            <div class="s-article-figure-01__row row">
+                <div class="s-article-figure-01__col s-article-figure-01__col--figure col-12 col-lg-4 d-flex align-items-center justify-content-center">
+                    <figure class="w-50 w-lg-100">
```

Comment:
J'ai un doute sur le seuil de rupture mobile/desktop : ça semble rentrer en colonne en `-md` ou alors peut-être réduire encore la largeur de l'illustration en dessous de `-lg` car dans certains cas elles prennent vraiment beaucoup de place. Qu'en penses-tu ?

| Actuellement <br /> ____________________ | Actuellement <br /> ____________________ |
|---|---|
| <img width="862" alt="Capture d’écran 2024-03-12 à 14 54 40" src="https://github.com/gip-inclusion/les-emplois/assets/705214/17a3a66b-fb4d-49f6-a3cd-b7f17f8a0910"> | <img width="856" alt="Capture d’écran 2024-03-12 à 14 55 20" src="https://github.com/gip-inclusion/les-emplois/assets/705214/544d002e-0f5c-429c-9527-55a7108f9dd3"> |

| Avec colonnes `-md` <br />(+ `w-md-75` en dessous) <br /> ____________________ | Avec figure `w-md-25` <br /> ____________________ |
|---|---|
| <img width="864" alt="Capture d’écran 2024-03-12 à 14 54 11" src="https://github.com/gip-inclusion/les-emplois/assets/705214/1d0989ab-1438-4728-be2c-3ca39d4e0e6f"> | <img width="856" alt="Capture d’écran 2024-03-12 à 15 06 42" src="https://github.com/gip-inclusion/les-emplois/assets/705214/28c7b9f6-4c9c-4692-bf4b-40d79529550d"> |
| <img width="859" alt="Capture d’écran 2024-03-12 à 15 12 17" src="https://github.com/gip-inclusion/les-emplois/assets/705214/9b370c5a-5103-4033-bf78-8fd3f092ffed"> | <img width="857" alt="Capture d’écran 2024-03-12 à 15 05 38" src="https://github.com/gip-inclusion/les-emplois/assets/705214/612edcd5-cf65-47b2-8b01-8165fcd51f42"> |

Reply:
Ok why not. Mais faut que je voie ce qu'en dit Antoine, car il avait validé comme ça.